docs/.docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+Contributing to Mayflower Documentation
+=========================
+
+Documentation for the overall Mayflower design system, including guiding principles hosted by Gitbook. 
+
+
+From Gitbook Editor:
+------------------------------------
+
+1. [Create](https://www.gitbook.com/join) a Gitbook account and ask for edit permission.
```

Comment:
@clairesunstudio --> do you want to indicate who can provide edit permissions here?